app/models/user.rb

```diff
@@ -73,6 +73,12 @@ class User < ActiveRecord::Base
 
   has_many :roles, :class_name => "UserRole"
 
+  has_many :issues, :class_name => "Issue", :foreign_key => :reported_user_id
+  has_one :issue, :class_name => "Issue", :foreign_key => :updated_by
```

Comment:
Does this make any sense? It's on `updated_by` so it's linking to issues updated by this user, but is `has_one` which doesn't seem to be right?